data/strings/types_strings.txt

```diff
@@ -16919,7 +16659,7 @@
     zh-Hant = 地產代理
 
   [type.office.government]
-    en = Government Office
+    en = Governmental Office
```

Comment:
Government office is fine.  governmental sounds wrong to me, or at least unusual/awkard.  (I'm native en_US in case that hasn't been clear.)

Reply:
i'll leave en as Government Office, but in the UK it's definitely Governmental office :)